assets/charts/components/metallb/templates/controller.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.controller.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "metallb.fullname" . }}-controller
```

Comment:
With this name change we will have two deployments of controller, no? Does helm take care of removing the cluster installed assets if those are removed from the chart?

Reply:
I verified it and it does remove the older deployments and daemonsets.

Reply:
That's the whole point of using Helm :)